src/containers/ProjectDashboardContainer.jsx

```diff
@@ -21,7 +21,7 @@ const propTypes = {
 
 class ProjectDashboardContainer extends Component {
 
-  componentDidMount() {
+  componentWillMount() {
```

Comment:
I didn't understand that argument either. `componentDidMount` would also be called twice, in that case. 

Is this worth discussing with the whole team? It sounds like something that affects everything we're working on, particularly zooniverse/Panoptes-Front-End